.scalafmt.conf

```diff
@@ -1,2 +1 @@
-include ".scalafmt-common.conf"
-project.includePaths = [] # disables formatting
+include ".scalafmt-local.conf"
```

Comment:
Whoever whants to enable scalafmt can just create a local `.scalafmt-local.conf` file with a single line containing `include ".scalafmt-common.conf"`.